.gitignore

```diff
@@ -11,3 +11,4 @@ var/log/rear
 .vscode
 servers
 /tools/run-in-docker-script.sh
+*.html
```

Comment:
can you please split this change into its own commit as it is unrelated to "Add support for new backup software Dell PowerProtect Data Manager"?

Reply:
Done